app/src/main/res/layout/fragment_life.xml

```diff
@@ -25,4 +25,4 @@
             android:textColor="@color/daynight_textColor" />
     </FrameLayout>
 
-</LinearLayout>
\ No newline at end of file
+</LinearLayout> 
```

Comment:
Is the extra space part of format?

Reply:
No, thanks for catching that @Kuljeet1998 ! removed the space